versioned_docs/version-3.0/ios-quickstart-manual.md

```diff
@@ -0,0 +1,325 @@
+---
+title: "Quickstart for integrating purchases with Adapty into your custom paywall in iOS SDK"
+description: "Quickstart guide to setting up Adapty for in-app subscription management."
+metadataTitle: "Adapty Quickstart Guide | Adapty Docs"
+keywords: ['paywalls ios', 'sdk ios', 'paywall', 'getPaywall']
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide shows you how to integrate Adapty's purchase handling into your custom paywall UI. You'll fetch products from Adapty, process purchases, and handle restores, while maintaining full control over your paywall design.
+
+:::important
+**This guide is for implementing custom paywalls.** If you want the easiest way to enable purchases, use the [Adapty Paywall Builder](ios-quickstart-paywalls.md). With Paywall Builder, you create paywalls in a no-code visual editor, and Adapty handles all purchase logic automatically.
+:::
+
+## Before you start
+
+### Set up products
+
+To enable in-app purchases, you need to understand three key concepts:
+
+- [**Products**](product.md) – anything users can buy (subscriptions, consumables, lifetime access)
+- [**Paywalls**](paywalls.md) are configurations that define which products to offer. In Adapty, paywalls are the only way to retrieve products, but this design lets you modify offerings, pricing, and product combinations without touching your app code.
+- [**Placements**](placements.md) – where and when you show paywalls in your app (like `main`, `onboarding`, `settings`). You set up paywalls for placements in the dashboard, then request them by placement ID in your code. This makes it easy to run A/B tests and show different paywalls to different users.
+
```

Comment:
1) Placements were probably the most confusing Adapty concept that I encountered so far, I think it warrants a more detailed explanation. I hope I didn't overdo it in the end.
2) The use of the word 'like' in this section is very colloquial, I'd avoid it

```suggestion
- [**Placements**](placements.md) – app locations with a paywall. Add placement code wherever you want to display paywalls: the settings menu, the premium content section, or anywhere else. Use the Adapty dashboard to connect your placements with your paywalls. When you invoke a placement, the Adapty servers determine which paywall to load. Adapty can display different paywalls to different user groups, making it easy to deploy regional strategies, or conduct A/B tests.
```